rma_account/models/rma_operation.py

```diff
@@ -17,3 +17,26 @@ class RmaOperation(models.Model):
         string="Refund Policy",
         default="no",
     )
+
+    refund_journal_id = fields.Many2one(
+        comodel_name="account.journal",
+        string="Refund Account Journal",
+        domain="[('id', 'in', valid_journal_ids)]",
+    )
+
+    valid_journal_ids = fields.One2many(
```

Comment:
valid_refund_journal_ids

Reply:
Also this should be a Many2many, it is compute and becouse of that doesn't fail, but strictly speaking this is many2many.